usagov/vote-gov-drupal
# VOTE-1078 Enable translation for paragraph types

## Jira ticket

[VOTE-1078](https://cm-jira.usa.gov/browse/VOTE-1078)

## Description

Enable translation of paragraph type fields.

## Deployment and testing

### Pre-deploy

1. lando retune

### QA/Test

1. Login to your local and edit a landing page (or make a new one)
2. Add all the paragraph types
3. Confirm that the fields for all paragraph types are translatable

## Checklist for the Developer




## Checklist for the Peer Reviewers
